src/vm/core.zig

```diff
@@ -1096,6 +1136,254 @@ test "trace is disabled" {
     }
 }
 
+test "deduceOp1 when opcode == .Call" {
+    // ************************************************************
+    // *                 SETUP TEST CONTEXT                       *
+    // ************************************************************
+    var allocator = std.testing.allocator;
+    var vm = try CairoVM.init(allocator, .{});
+    defer vm.deinit();
+
+    // ************************************************************
+    // *                      TEST BODY                           *
+    // ************************************************************
+    const instruction = instructions.Instruction{
```

Comment:
We are repeating this instruction for every test - we can also bring this into the global scope and change its values for each test. Something like:

```zig
var instruction.= Instruction { ... }

test "deduceOp1 when opcode == .Call" {
    instruction.opcode = .Call;
 
    // proceed with test
}
```